src/index.html

```diff
@@ -14,16 +14,16 @@ <h1>Leaderbourd</h1>
         <div class="bourd-countainer">
             <div class="bourd-header">
                 <h2>Recent scores</h2>
-                <button type="button">Referesh</button>
+                <button type="button" id="referesh">Referesh</button>
             </div>
             <ul class="scores"></ul>
         </div>
         <div class="form-container">
             <h2>Add score</h2>
-            <form action="submite" >
-                <input type="text" placeholder="Your name">
-                <input type="text" name="score" placeholder="Your score">
-                <button type="submit">Submite</button>
+            <form action="submite" id="form" >
+                <input type="text" placeholder="Your name" id="user-name" required>
+                <input type="text" name="score" placeholder="Your score" id="user-score" required>
+                <button type="submit" id="submit">Submite</button>
```

Comment:
- [ ] Kindly try to fix this typos error 😁 you may write **Submit** instead of **submite**, you may also write **Refresh** instead of **Referesh** to make your project professional 🙏

Reply:
Thank you for your insightful review 💯 .
required changes are implemmented.